### PR TITLE
Facebook Custom Audiences (Actions) documentation

### PR DESCRIPTION
**Pull request by forstisabella**
### Proposed changes
Add documentation for FCAA destination

### Merge timing
8/9

### CODE REVIEW

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+This destination sends audiences, or lists of users, from your data warehouse to Facebook Custom Audiences using Reverse ETL. Syncs can add or remove users from your Facebook Custom Audiences as records are created, updated, or deleted in your warehouse. 
+
+> info "This destination only supports **Reverse ETL sources**
+> This destination only supports [Reverse ETL sources](/docs/connections/reverse-etl/#what-warehouse-data-sources-does-segment-support). To connect Facebook Custom Audiences to Twilio Engage, use the [Facebook Custom Audiences](/docs/connections/destinations/catalog/personas-facebook-custom-audiences/) destination. 
+
+## Getting started
+
+### Prerequisites
+- A Reverse ETL source already set up. If you don't yet have a Reverse ETL source, follow the instructions in Segment's [Reverse ETL documentation](/docs/connections/reverse-etl/#getting-started). Segment recommends setting your External ID as your primary key for your Reverse ETL model, as you'll need an External ID to remove users from your custom audiences. 
+- A Facebook Custom Audiences account with the following permissions:
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:20]: @marinhero @mgutten - do you all know which permissions (if any) someone would need?

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+---
+title: Facebook Custom Audiences (Actions)
+id: 645d5fc12eb891cf0a93fe4b
+hidden: true
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:4]: if this is going into public beta we should remove this line

```suggestion
```

**Comment by nick-Ag** [comment]
Hey @forstisabella - I've added some code to hide certain legacy actions which will soon be removed, I also removed the `hidden: true` from the entire destination. I don't see those changes showing up in the netlify preview however. Do you know what's going wrong there?

**Comment by forstisabella** [comment]
hi @nick-Ag - it looks like you used the [netlify-build] format correctly and the builds are working on the back end, however the destination info box at the top of the page and the Available Actions section only build after we pull the catalog from the public API as part of the deploy process, so we won't see those changes until we are ready to deploy the docs!

**Comment by forstisabella** [comment]
I do have to add this destination to the Facebook strat nav, which may be some of what you're seeing! brb

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  <li value="6" markdown=1>
+  Set how often your model syncs by setting the [Sync schedule](/docs/connections/reverse-etl/#step-4-create-mappings). 
+  </li>
+  <li value="7" markdown=1>
+  Select or create an audience in Facebook to sync your data with. Click the **Select or create audience in Facebook** button to save the audience ID to your mapping. 
+  </li>
+  <li value="8" markdown=1>
+  Map your model columns to the appropriate Facebook Custom Audience parameters. For more context about data formatting, see the [Sync audience](#sync-audience) and [Data processing](#data-processing) documentation. 
+  <ul> 
+    <li> Map External ID to a unique user identifier from your system (like User ID, CRM ID, or anonymous ID.) Segment recommends using the External ID column as your primary key when setting up your Reverse ETL model so you can more easily remove users from your custom audience. External ID is the only field Segment requires you to map. </li>
```
Comment (mgutten) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:64]: "...only field Segment requires you to map..." -> "...only field Facebook requires."

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+5. Under **Select record to map and send**, select **Deleted records**. The Deleted records sync mode removes any records from your custom audience that you deleted from your source model.
+6. Set how often your model syncs by setting the [Sync schedule](/docs/connections/reverse-etl/#step-4-create-mappings). 
+7. Select or create an audience in Facebook to sync your data with. Click the **Select or create audience in Facebook** button to save the audience ID to your mapping.
+8. Map your model columns to the appropriate Facebook Custom Audience parameters. For more context about data formatting, see the [Sync audience](#sync-audience) and [Data processing](#data-processing) documentation. Map your External ID to a unique user identifier from your system (like User ID, CRM ID, or anonymous ID.) Segment recommends using the External ID column as your primary key when setting up your Reverse ETL model so you can more easily remove users from your custom audience. External ID is the only field Segment requires you to map.
```
Comment (mgutten) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:85]: "For more context about data formatting, see the [Sync audience](#sync-audience) and [Data processing](#data-processing) documentation. Map your External ID to a unique user identifier from your system (like User ID, CRM ID, or anonymous ID.) Segment recommends using the External ID column as your primary key when setting up your Reverse ETL model so you can more easily remove users from your custom audience. External ID is the only field Segment requires you to map."

should be:

"Only the External ID is required. When a record is deleted from your source model, only the model primary key is sent to the mapping; other columns from your source model are not sent. We highly recommend using the External ID as your primary key in your source model."

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
+---
+title: Facebook Custom Audiences (Actions)
+id: 645d5fc12eb891cf0a93fe4b
+beta: true
```
[the remark this answers is not in the document]
Comment (forstisabella) [reply src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:4]: @mgutten This is a feature, not a bug! For destinations in Private Beta, this box doesn't render. When the destination is moved into Public Beta tomorrow and the docs catalog updates, the box will render correctly!

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+This destination sends audiences, or lists of users, from your data warehouse to Facebook Custom Audiences using Reverse ETL. Syncs can add or remove users from your Facebook Custom Audiences as records are created, updated, or deleted in your warehouse. 
+
+> info "This destination only supports **Reverse ETL sources**"
+> This destination only supports [Reverse ETL sources](/docs/connections/reverse-etl/#what-warehouse-data-sources-does-segment-support). To connect Facebook Custom Audiences to Twilio Engage, use the [Facebook Custom Audiences](/docs/connections/destinations/catalog/personas-facebook-custom-audiences/) destination. 
+
+## Getting started
+
+### Prerequisites
+- A Reverse ETL source already set up. If you don't yet have a Reverse ETL source, follow the instructions in Segment's [Reverse ETL documentation](/docs/connections/reverse-etl/#getting-started). Segment recommends setting your External ID as your primary key for your Reverse ETL model, as you'll need an External ID to remove users from your custom audiences.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:27]:
```suggestion
- A Reverse ETL source already set up. If you don't yet have a Reverse ETL source, follow the instructions in Segment's [Reverse ETL documentation](/docs/connections/reverse-etl/#getting-started). Segment recommends setting your [External ID](/docs/connections/destinations/catalog/actions-facebook-conversions-api/#destination-settings) as the primary key for your Reverse ETL model, as you'll need an External ID to remove users from your custom audiences. 
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+  <li value="6" markdown=1>
+  Set how often your model syncs by setting the [Sync schedule](/docs/connections/reverse-etl/#step-4-create-mappings). 
+  </li>
+  <li value="7" markdown=1>
+  Select or create an audience in Facebook to sync your data with. Click the **Select or create audience in Facebook** button to save the audience ID to your mapping. 
+  </li>
+  <li value="8" markdown=1>
+  Map your model columns to the appropriate Facebook Custom Audience parameters. For more context about data formatting, see the [Sync audience](#sync-audience) and [Data processing](#data-processing) documentation. 
+  <ul> 
+    <li> Map External ID to a unique user identifier from your system (like User ID, CRM ID, or anonymous ID.) Segment recommends using the External ID column as your primary key when setting up your Reverse ETL model so you can more easily remove users from your custom audience. External ID is the only field Segment requires you to map. </li>
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:64]:
```suggestion
    <li> Map External ID to a unique user identifier from your system (like User ID, CRM ID, or anonymous ID.) Segment recommends using the External ID column as your primary key when setting up your Reverse ETL model so you can more easily remove users from your custom audience. External ID is the only field Facebook requires. </li> 
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+<ol style="counter-reset: none;">
+  <li value="6" markdown=1>
+  Set how often your model syncs by setting the [Sync schedule](/docs/connections/reverse-etl/#step-4-create-mappings). 
+  </li>
+  <li value="7" markdown=1>
+  Select or create an audience in Facebook to sync your data with. Click the **Select or create audience in Facebook** button to save the audience ID to your mapping. 
+  </li>
+  <li value="8" markdown=1>
+  Map your model columns to the appropriate Facebook Custom Audience parameters. For more context about data formatting, see the [Sync audience](#sync-audience) and [Data processing](#data-processing) documentation.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:62]:
```suggestion
  Map your model columns to the appropriate Facebook Custom Audience parameters. For more context about data formatting, see the [Sync Audience](#sync-audience) and [Data processing](#data-processing) documentation. 
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+5. Under **Select record to map and send**, select **Deleted records**. The Deleted records sync mode removes any records from your custom audience that you deleted from your source model.
+6. Set how often your model syncs by setting the [Sync schedule](/docs/connections/reverse-etl/#step-4-create-mappings). 
+7. Select or create an audience in Facebook to sync your data with. Click the **Select or create audience in Facebook** button to save the audience ID to your mapping.
+8. Map your model columns to the appropriate Facebook Custom Audience parameters. For more context about data formatting, see the [Sync audience](#sync-audience) and [Data processing](#data-processing) documentation. Map your External ID to a unique user identifier from your system (like User ID, CRM ID, or anonymous ID.) Segment recommends using the External ID column as your primary key when setting up your Reverse ETL model so you can more easily remove users from your custom audience. External ID is the only field Segment requires you to map.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:85]:
```suggestion
8. Map your model columns to the appropriate Facebook Custom Audience parameters. Only the External ID is required. When a record is deleted from your source model, only the model primary key is sent to the mapping; other columns from your source model are not sent. Segment recommends using the External ID as your primary key in your source model.
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md (hunk 1 above)
Comment (mgutten) [reply src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:28]: Let's update this to:

- A Facebook account with the following permissions for the target Facebook Ads Account(s):
- [ads_management](https://developers.facebook.com/docs/permissions#ads_management)

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+* City
+* State
+* Country
+
+Segment normalizes the Phone field by removing any non-numeric symbols, whitespace, and leading zeroes.
+
+### Hashing
+Facebook requires you to hash all PII before sending it to the Facebook Conversions API. 
+
+Segment hashes the following fields by default:
```
Comment (mgutten) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:110]: Update to:

"Segment will automatically hash any of the following fields that are not already SHA256 hashed at egress:"

Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:110]:
```suggestion
Segment automatically hashes any of the following fields that are not already SHA256 hashed at egress:
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+5. Under **Select record to map and send**, select **Deleted records**. The Deleted records sync mode removes any records from your custom audience that you deleted from your source model.
+6. Set how often your model syncs by setting the [Sync schedule](/docs/connections/reverse-etl/#step-4-create-mappings). 
+7. Select or create an audience in Facebook to sync your data with. Click the **Select or create audience in Facebook** button to save the audience ID to your mapping.
+8. Map your model columns to the appropriate Facebook Custom Audience parameters. Only the External ID is required. When a record is deleted from your source model, only the model primary key is sent to the mapping; other columns from your source model are not sent. Segment recommends using the External ID as your primary key in your source model.
+9. Send a test record. If successful, you should see a 200 response in Segment and one record removed from your custom audience. To verify that the record was successfully removed from your custom audience, open Facebook Ads Manager and navigate to **Audiences > {Audience Name} > History**.
```
Comment (pwseg) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:86]:
```suggestion
9. Send a test record. If successful, you should see a `200` response in Segment and one record removed from your custom audience. To verify that the record was successfully removed from your custom audience, open Facebook Ads Manager and navigate to **Audiences > {Audience Name} > History**.
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+> info "This destination only supports Reverse ETL sources"
+> This destination only supports [Reverse ETL sources](/docs/connections/reverse-etl/#what-warehouse-data-sources-does-segment-support). To connect Facebook Custom Audiences to Twilio Engage, use the [Facebook Custom Audiences](/docs/connections/destinations/catalog/personas-facebook-custom-audiences/) destination. 
+
+## Getting started
+
+### Prerequisites
+- A Reverse ETL source already set up. If you don't yet have a Reverse ETL source, follow the instructions in Segment's [Reverse ETL documentation](/docs/connections/reverse-etl/#getting-started). Segment recommends setting an [External ID](#destination-settings) as the primary key for your Reverse ETL model, as you'll need an External ID to remove users from your custom audiences. 
+- A Facebook Custom Audiences account with the following permissions: 
+  -
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:29]:
```suggestion
- A Facebook account with [ads_management](https://developers.facebook.com/docs/permissions#ads_management){:target="_blank”} permissions for the target Facebook Ads Account(s).
```

File: src/_data/sidenav/strat.yml [hunk 13]
```diff
@@ -44,6 +44,8 @@ sections:
     title: Facebook Ads cloud source
   - path: /connections/sources/catalog/cloud-apps/facebook-lead-ads
     title: Facebook Lead Ads cloud source
+  - path: /connections/destinations/catalog/actions-facebook-custom-audiences
+    title: Facebook Custom Audiences (Actions) destinations
```
Comment (forstisabella) [inline src/_data/sidenav/strat.yml:48]:
```suggestion
  - path: /connections/destinations/catalog/actions-facebook-custom-audiences
    title: Facebook Custom Audiences (Actions) destinations
```
```suggestion
  - path: /connections/destinations/catalog/actions-facebook-custom-audiences
    title: Facebook Custom Audiences (Actions) destination
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+
+This destination sends audiences, or lists of users, from your data warehouse to Facebook Custom Audiences using Reverse ETL. Syncs can add or remove users from your Facebook Custom Audiences as records are created, updated, or deleted in your warehouse. 
+
+> info "This destination only supports **Reverse ETL sources**
+> This destination only supports [Reverse ETL sources](/docs/connections/reverse-etl/#what-warehouse-data-sources-does-segment-support). To connect Facebook Custom Audiences to Twilio Engage, use the [Facebook Custom Audiences](/docs/connections/destinations/catalog/personas-facebook-custom-audiences/) destination. 
+
+## Getting started
+
+### Prerequisites
+- A Reverse ETL source already set up. If you don't yet have a Reverse ETL source, follow the instructions in Segment's [Reverse ETL documentation](/docs/connections/reverse-etl/#getting-started). Segment recommends setting your External ID as your primary key for your Reverse ETL model, as you'll need an External ID to remove users from your custom audiences.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md]:
```suggestion
- A Reverse ETL source already set up. If you don't yet have a Reverse ETL source, follow the instructions in Segment's [Reverse ETL documentation](/docs/connections/reverse-etl/#getting-started). Segment recommends setting your [External ID](#available-actions) as your primary key for your Reverse ETL model, as you'll need an External ID to remove users from your custom audiences. 
```

File: src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+  </li>
+  <li value="8" markdown=1>
+  Map your model columns to the appropriate Facebook Custom Audience parameters. For more context about data formatting, see the [Sync Audience](#sync-audience) and [Data processing](#data-processing) documentation. 
+  <ul> 
+    <li> Map External ID to a unique user identifier from your system (like User ID, CRM ID, or anonymous ID.) Segment recommends using the External ID column as your primary key when setting up your Reverse ETL model so you can more easily remove users from your custom audience. External ID is the only field Facebook requires. </li> 
+    <li> Segment recommends mapping as many parameters as you have available in your source model so that you can increase your match rates. </li> 
+  </ul>
+  </li>
+  <li value ="9" markdown=1>
+  Send a test record. If successful, you should see a 200 response in Segment and one added record to your custom audience. To verify that the record was successfully added to your custom audience, open Facebook Ads Manager and navigate to **Audiences > {Audience Name} > History**.
```
Comment (pwseg) [inline src/connections/destinations/catalog/actions-facebook-custom-audiences/index.md:68]:
```suggestion
  Send a test record. If successful, you should see a `200` response in Segment and one added record to your custom audience. To verify that the record was successfully added to your custom audience, open Facebook Ads Manager and navigate to **Audiences > {Audience Name} > History**.
```

**Review by pwseg — APPROVED** [review]

**Pull request merged**